src/blocktypes/installables.nix

```diff
@@ -45,6 +47,7 @@
           nix profile remove ${flake}#${fragment}
         '';
       }
+      # TODO: use target. `nix bundle` requires a flake ref, but we may be able to use nix-bundle instead as a workaround
```

Comment:
seems worthy of an upstream issue

Reply:
agreed

Reply:
https://github.com/NixOS/nix/issues/7481